src/duckling/game/drawable/container-drawable.component.ts

```diff
@@ -57,6 +59,17 @@ export class ContainerDrawableComponent {
     @Input() containerDrawable : ContainerDrawable;
     @Output() drawableChanged = new EventEmitter<ContainerDrawable>();
 
+    onChildDrawableCloned(newDrawables : Drawable[]) {
```

Comment:
There is a bug here. If you have a container in a container and you clone it, then the children of the container will have duplicate keys. You should fix that for this and animated drawable.